robot:
  SDK_version: 2
  robot_name: "HuangPx HECTOR"
  arm_dof: 0
  leg_dof: 6
  # Serial_Type: "/dev/ttyUSB"  # stm32的
  Serial_Type: "/dev/ttyACM"  # 5361的
  Seial_baudrate: 4000000
  Serial_allocate: "1for2" #一个ft4232带2块CANboard,此时默认一块板子引出2个串口,一个串口与一个CAN口相对应
                #  "1for1"  一个ft4232带1块CANboard,此时默认一块板子引出4个串口,一个串口与一个CAN口相对应
  CAN_Type: "CAN-FD BRS" # "CAN-FD"\"CAN-FD BRS"\"CAN 2.0B"
  control_type: 9  # 运行模式
  CANboard_type: "STM32 H730"
  CANboard_num: 1
  CANboard:
    No_1_CANboard:
      CANport_num: 2
      CANport:
        CANport_1:
          motor_num: 6
          motor:
            motor1:
                type: 3
                id: 1
                name: "L_low_foot"
                num: 1
                pos_limit_enable: false
                pos_upper: 5
                pos_lower: -5
                tor_limit_enable: false
                tor_upper: 5
                tor_lower: -3
            motor2:
                type: 3
                id: 2
                name: "L_up_foot"
                num: 2
                pos_limit_enable: false
                pos_upper: 5
                pos_lower: -5
                tor_limit_enable: false
                tor_upper: 5
                tor_lower: -3
            motor3:
                type: 3
                id: 3
                name: "L_calf"
                num: 3
                pos_limit_enable: false
                pos_upper: 5
                pos_lower: -5
                tor_limit_enable: false
                tor_upper: 5
                tor_lower: -3
            motor4:
                type: 3
                id: 4
                name: "L_hip_pitch"
                num: 4
                pos_limit_enable: false
                pos_upper: 5
                pos_lower: -5
                tor_limit_enable: false
                tor_upper: 5
                tor_lower: -3
            motor5:
                type: 3
                id: 5
                name: "L_hip_roll"
                num: 5
                pos_limit_enable: false
                pos_upper: 5
                pos_lower: -5
                tor_limit_enable: false
                tor_upper: 5
                tor_lower: -3
            motor6:
                type: 3
                id: 6
                name: "L_hip_yaw"
                num: 6
                pos_limit_enable: false
                pos_upper: 5
                pos_lower: -5
                tor_limit_enable: false
                tor_upper: 5
                tor_lower: -3
        CANport_2:
          motor_num: 6
          motor:
            motor1:
                type: 3
                id: 1
                name: "R_low_foot"
                num: 1
                pos_limit_enable: false
                pos_upper: 5
                pos_lower: -5
                tor_limit_enable: false
                tor_upper: 5
                tor_lower: -3
            motor2:
                type: 3
                id: 2
                name: "R_up_foot"
                num: 2
                pos_limit_enable: false
                pos_upper: 5
                pos_lower: -5
                tor_limit_enable: false
                tor_upper: 5
                tor_lower: -3
            motor3:
                type: 3
                id: 3
                name: "R_calf"
                num: 3
                pos_limit_enable: false
                pos_upper: 5
                pos_lower: -5
                tor_limit_enable: false
                tor_upper: 5
                tor_lower: -3
            motor4:
                type: 3
                id: 4
                name: "R_hip_pitch"
                num: 4
                pos_limit_enable: false
                pos_upper: 5
                pos_lower: -5
                tor_limit_enable: false
                tor_upper: 5
                tor_lower: -3
            motor5:
                type: 3
                id: 5
                name: "R_hip_roll"
                num: 5
                pos_limit_enable: false
                pos_upper: 5
                pos_lower: -5
                tor_limit_enable: false
                tor_upper: 5
                tor_lower: -3
            motor6:
                type: 3
                id: 6
                name: "R_hip_yaw"
                num: 6
                pos_limit_enable: false
                pos_upper: 5
                pos_lower: -5
                tor_limit_enable: false
                tor_upper: 5
                tor_lower: -3
